Cql/Cql.Compiler/Extensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Hl7.Cql.Elm;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Hl7.Cql.Compiler
+{
+    internal static class Extensions
+    {
+        public static string OrEmptyString(this string? value)
```

Comment:
All of these methods are unnecessary and reduce performance.  Every C# programmer can read `x ?? ""` as "x or empty string" without a method name spelling it out.

Reply:
Got rid of this class